docs/static/releasenotes.asciidoc

```diff
@@ -48,6 +49,193 @@ This section summarizes the changes in the following releases:
 * <<logstash-7-0-0-alpha2,Logstash 7.0.0-alpha2>>
 * <<logstash-7-0-0-alpha1,Logstash 7.0.0-alpha1>>
 
+[[logstash-7-14-0]]
+=== Logstash 7.14.0 Release Notes
+
+[[featured-7-14-0]]
+==== New features and enhancements
+
+[[agent-7-14-0]]
+===== Elastic Agent input plugin
+
+The {ls} <<plugins-inputs-elastic_agent,Elastic Agent input plugin>> is now available to coincide with Elastic Agent's general availability. 
+The new plugin is based on the <<plugins-inputs-beats,Beats input plugin>>, and will seem familiar to users who have been using the Beats input.
+
+[[entsearch-7-14-0]]
+===== Elastic Enterprise Search integration
+
+The new <<plugins-outputs-elastic_workplace_search,Elastic Workplace Search>> plugin is available as part of the <<plugins-integrations-elastic_enterprise_search,Elastic Enterprise Search integration>> and is bundled with {ls} 7.14.0. 
+The output plugins in this integration send events from {ls} to https://www.elastic.co/enterprise-search[Elastic Enterprise Search].
+
+[[geoip-7-14-0]]
+===== MaxMind GeoIP database changes
+
+MaxMind, the company that provides the GeoIP databases that {ls} uses, has changed their licensing. 
+Logstash has made changes to the core product and the <<plugins-filters-geoip,GeoIP filter plugin>> to support these changes and to help you manage your geoip databases and updates. 
+See <<plugins-filters-geoip-database_license>> and <<plugins-filters-geoip-database_auto>> for more details.
+
+<<plugins-filters-geoip-metrics,Geoip database metrics>> are now available in /node/stats API.
+
+[[ecs-7-14-0]]
+===== Progress toward Elastic Common Schema (ECS)
+In this release, we've made more Logstash plugins compatible with the Elastic Common Schema (ECS):
+
+* <<plugins-filters-csv,csv filter>>
+* <<plugins-filters-fingerprint,fingerprint filter>>
+* <<plugins-filters-jdbc_static,jdbc_static filter>>
+* <<plugins-filters-json,json filter>>
+* <<plugins-filters-useragent,useragent filter>>
+* <<plugins-inputs-file,file input>>
+* <<plugins-inputs-http,http input>>
+* <<plugins-inputs-jdbc,jdbc input>>
+* <<plugins-inputs-s3,s3 input>>
+* <<plugins-inputs-tcp,tcp input>>
+
+Values for `pipeline.ordered` and `pipeline.ecs_compatibility` settings are now supported for Logstash Central Management in Kibana. https://github.com/elastic/logstash/pull/12861[#12861]
```

Comment:
IIRC, we added one layer so that if the settings were present in the pipeline's definition we would use them correctly, but there is another layer in Kibana itself that will be necessary to make it so that users can actually _set_ these settings in Central Management.